Gemfile.lock

```diff
@@ -98,4 +100,4 @@ DEPENDENCIES
   xcpretty
 
 BUNDLED WITH
-   1.17.3
+   2.2.23
```

Comment:
Did you manually test that things still work OK after this update? I know this repo has no CI & limited testing (if any)

Reply:
Ah no didn't realize there wasn't CI... I actually am not able to build the visa-checkout demo on master or this branch (Maybe because I am on Xcode 12.5?) I'll have to look more into this.

Reply:
It _should_ work with all versions of XCode 12, I think the last time we checked was around XCode 12.3.